src/transactions/OperationFrame.cpp

```diff
@@ -15,6 +15,7 @@
 #include "transactions/CreatePassiveSellOfferOpFrame.h"
 #include "transactions/EndSponsoringFutureReservesOpFrame.h"
 #include "transactions/InflationOpFrame.h"
+#include "transactions/InvokeHostFunctionOpFrame.h"
```

Comment:
Not necessary since InvokeHostFunctionOpFrame.h itself is wrapped in an ifdef, but should we add the conditional guard here as well to be thorough?